### PR TITLE
Remove body font normalization on contact-form module

### PR DESCRIPTION
**Pull request by CGastrell**
This PR addresses an issue seen on classic editor, where the entire body style would be "normalized" to use sans-serif fonts. A user came complaining he was not happy with it. In the end, there is no reason why the module/package should normalize something at such a high level.

## Proposed changes:
Remove body font normalization from CSS files.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion
p1677226003014399-slack-CBG1CP4EN

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Install classic editor plugin, have Jetpack plugin enabled. Create a post and verify the editor fonts remain as usual (default is some serif font Georgia, Times New Roman... )

### CODE REVIEW

**Review by anomiex — COMMENTED** [review]
While this looks like it will fix the specific issue referenced, by removing the setting of the font on `body`, I note it stops short of doing what it says it's doing since there is still a bunch of other top-level styling going on.

Since I'm guessing you don't want to rework all the styles here to scope them to an appropriate container element, maybe just adjust the changelog entry and PR title to refer to the body font rather than all the CSS normalization?

**Comment by CGastrell** [comment]
> While this looks like it will fix the specific issue referenced, by removing the setting of the font on `body`, I note it stops short of doing what it says it's doing since there is still a bunch of other top-level styling going on.
> 
> Since I'm guessing you don't want to rework all the styles here to scope them to an appropriate container element, maybe just adjust the changelog entry and PR title to refer to the body font rather than all the CSS normalization?

The changelog entry is purposely specific about this: "Remove CSS normalization for body on contact-form module and package"

As for PR naming, I can change it.

Regarding a more comprehensive removal of _CSS normalization_, I'd go cautiously about it. People has been using the module and, inadvertently, its CSS normalization for quite some time. While we want to improve the overall code of the module, even turning it into its own package, unsolicitedly removing CSS might turn into backlash/complaints as that would affect already working/published posts and forms.

**Comment by anomiex** [comment]
But there are still a bunch of styles being set on the body:
https://github.com/Automattic/jetpack/blob/de5786b817e678690110ec966a9b4ed69b31703a/projects/packages/forms/src/contact-form/css/editor-inline-editing-style.css#L20-L34

File: projects/packages/forms/changelog/fix-classic-editor-styles [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fixed
+
+Remove CSS normalization for body on contact-form module and package
```
Comment (anomiex) [inline projects/packages/forms/changelog/fix-classic-editor-styles]:
```suggestion
Remove CSS normalization for body fonts on contact-form module and package
```

File: projects/plugins/jetpack/changelog/fix-classic-editor-styles [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: bugfix
+
+Remove CSS normalization for body on contact-form module and package
```
Comment (anomiex) [inline projects/plugins/jetpack/changelog/fix-classic-editor-styles]:
```suggestion
Remove CSS normalization for body fonts on contact-form module and package
```

**Review by anomiex — APPROVED** [review]

**Pull request merged**